### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by VladimirtheGreatest**
Masterpiece6 + additonal changes and queries for mobile navigation.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,2451 @@
...
+  /* make it invisible til hover but set the styling */
+  .dropdown-content {
+    display: none;
+    position: absolute;
+    background-color: #2a2a2a;
+    min-width: 180px;
+    max-width: 180px;
+    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+    z-index: 1;
+    right: 210px;
```
Comment (stephairth) [inline css/style.css]: change "right: 210px" to "right: 160px" to re-align the language dropdown, please!

**Review by stephairth — COMMENTED** [review]
Please change "right: 210px" at line 1743 to "right: 160px"--otherwise looks good! Is there any way we can make the images in main section 2 less stretched?

**Review by stephairth — APPROVED** [review]
compressed images look good to me

**Comment by sasigit7** [comment]
> Please change "right: 210px" at line 1743 to "right: 160px"--otherwise looks good! Is there any way we can make the images in main section 2 less stretched?

**Comment by sasigit7** [comment]
Pheww!! Apologies!! By mistake, I closed the pull request. Reopening it.

File: css/style.css (hunk 1 above)
Comment (VladimirtheGreatest) [reply css/style.css]: changed

**Pull request merged**